Microsoft.Toolkit.Win32.UI.Controls/WPF/WebView/WebView.cs

```diff
@@ -0,0 +1,1046 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows;
+using Microsoft.Toolkit.Win32.UI.Controls.Interop.Win32;
+using Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT;
+using Microsoft.Toolkit.Win32.UI.Controls.WinForms;
+using Point = Windows.Foundation.Point;
+using Size = Windows.Foundation.Size;
+
+/*
+
+Overview of Keyboard Input Routing for IWebViewControl
+
+The window hosting IWebViewControl receives regular alphanumeric keyboard input via its WndProc, and whoever is running the message loop
+calls DispatchMessage() after any preprocessing and special routing, and the message gets to the WndProc of the window with focus. "Accelerator"
+keys need to be passed to the IWebViewControl via TranslateAccelerator()
+
+However, due to the way the window is hosted out of process, we are not able to intercept those messages. As such we have no support to handle
+tabbing in or out of the IWebBrowserControl window.
+
+NOTE: Depending on how the control is hosted, input messages flow different
+
+ */
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.WPF
+{
+    /// <summary>
+    /// This class is an implementation of <see cref="IWebView"/> for WPF. This class cannot be inherited.
+    /// </summary>
+    /// <seealso cref="Microsoft.Toolkit.Win32.UI.Controls.WPF.WebViewHost" />
+    /// <seealso cref="Microsoft.Toolkit.Win32.UI.Controls.IWebView" />
+    public sealed class WebView : WebViewHost, IWebView
```

Comment:
@normesta @nmetulev  Most documentation is in `IWebView,` and I've been switching documentation over to use `<inheritdoc />` with a VS plugin that updates the generated XML documentation. There's a NuGet package available that I think we should use, so I don't have to duplicate documentation all over the place and keep it up to date.